core/src/com/unciv/logic/civilization/TechManager.kt

```diff
@@ -103,8 +114,7 @@ class TechManager {
     }
 
     fun nextTurn(scienceForNewTurn: Int) {
-        val currentTechnology = currentTechnologyName()
-        if (currentTechnology == null) return
+        val currentTechnology = currentTechnologyName() ?: return
```

Comment:
Please don't use the Elvis operator, it's less understandable for people who aren't used to it